mlir/test/lib/Dialect/Test/TestOpsSyntax.td

```diff
@@ -568,6 +568,8 @@ def FormatLiteralFollowingOptionalGroup
   let assemblyFormat = "(`(` $value^ `)`)? `:` $type attr-dict";
 }
 
+
+
```

Comment:
Remove? This file is unchanged otherwise

Reply:
Done.